apps/gateway/docker/Dockerfile

```diff
@@ -1,64 +1,73 @@
-FROM node:20-alpine AS base
+ARG PROJECT="@latitude-data/gateway"
+ARG PROJECT_PATH="apps/gateway"
```

Comment:
both gateway and workers have the same dockerfile and you only have to change these two args for them to compile